templates/glossary-body-all.html

```diff
@@ -0,0 +1,117 @@
+<style>
+  #divAlphaTerms {
+    -webkit-column-count: 3;
+    /* -moz-column-count: 3; */
+    column-count: 3;
+    float: left;
+    /* width: 33.33%; */
+  }
+
+  #divAlphaTerms ul::before {
+    content: attr(data-ref) " ";
+    font-weight: 900;
+    font-size: 2.07rem;
+    line-height: 3.43rem;
+    display: block;
+    border-bottom: 1px solid #888;
+  }
+
+  alphabet-button {
+    margin: 5px;
+  }
+</style>
+
+<html>
```

Comment:
the `html`, `head` and `body` is included in the `$OUTPUT->header(); ... $OUTPUT->bodyStart();`

No need to do it again.